Android/JaiJaveria/Beginner/SplashScreen/app/src/main/java/com/example/splashscreen/SliderAdapter.java

```diff
@@ -0,0 +1,72 @@
+package com.example.splashscreen;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.viewpager.widget.PagerAdapter;
+
+public class SliderAdapter extends PagerAdapter {
+    Context context;
+    LayoutInflater li;
+    public SliderAdapter(Context context)
+    {
+        this.context=context;
+    }
+    //Arrays
+//    images taken from pixabay.com
+    public int[] slide_images= {
+
+            R.drawable.virus,
+            R.drawable.handwash,
+            R.drawable.social_dis
+    };
+    public static String[] slide_headings={
+        "Pandemic",
```

Comment:
Place the strings in the resource file and access them.

Reply:
the getString() methods doesnt seem to be working. One needs to extend an fragment/activity class to do so but my class already extends PagerAdapter so I cannot do that as well. Is there any other way to do so?